### PR TITLE
New Line_Flowering Parameter

### PR DESCRIPTION
**Pull request by wutermeloon**
Some new parameters have been added, stop_at_end and min_speed. Also, some parameters have been given default values so that they don't take up as much space.

### CODE REVIEW

File: Traffic_Crane.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    m.on_for_seconds(-5, 20, 2.5)
+    # go back
+    m.on(-5, -20)
+    while is_right_white() != True:
+        pass
+    m.off()
+    #Turning towards crane mission
+    m.on(30, -30)
+    #Stoping at line connected to base
+    while is_left_black() != True:
```
Comment (Butterspot) [inline Traffic_Crane.py]: Can we change this to "is_left_not_black" to make it simpler?

File: color.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ def is_left_black():
         return False
 
 def is_left_not_white():
-    if left_color_sensor_rli() < 90:
```
Comment (Butterspot) [inline color.py:32]: If the reflection value is 50, should it be not black, or not white?

File: Traffic_Crane.py (hunk 1 above)
Comment (wutermeloon) [reply Traffic_Crane.py]: I have changed the "not_white" to "other_shade" to say it is not black or white.

File: color.py (hunk 2 above)
Comment (wutermeloon) [reply color.py:32]: The value 50 in this program area is to determine that the value is not white and is most likely the other colors on the map.

**Review by Butterspot — APPROVED** [review]

**Pull request merged**